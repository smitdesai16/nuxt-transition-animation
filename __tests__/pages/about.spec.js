import { shallowMount } from '@vue/test-utils'
import About from '@/pages/about'

const factory = () => {
	return shallowMount(About, {
	})
}

describe('about', () => {
	test('mounts properly', () => {
		const wrapper = factory()
		expect(wrapper.isVueInstance()).toBeTruthy()
	})

	test('renders properly', () => {
		const wrapper = factory()
		expect(wrapper.html()).toMatchSnapshot()
	})

	test('h1 renders properly', () => {
		const wrapper = factory()
		expect(wrapper.find('h1').text()).toBe('About')
	})
})
