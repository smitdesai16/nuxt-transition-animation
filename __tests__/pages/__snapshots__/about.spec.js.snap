// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`about renders properly 1`] = `
"<div class=\\"container-xl\\">
  <div class=\\"row\\">
    <div class=\\"col-12\\">
      <h1>
        About
      </h1>
    </div>
  </div>
</div>"
`;
